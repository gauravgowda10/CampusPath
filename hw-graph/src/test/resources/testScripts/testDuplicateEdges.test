# Tests that adding the same edges multiple times

# Create graph
CreateGraph duplicateEdges

# Add nodes to duplicateEdges
AddNode duplicateEdges x
AddNode duplicateEdges y

# Add edges to duplicateEdges
AddEdge duplicateEdges x x sameXX
AddEdge duplicateEdges y y sameYY
AddEdge duplicateEdges x y diffXY1

# Add Duplicate edges to duplicateEdges
AddEdge duplicateEdges x x sameXX
AddEdge duplicateEdges y y sameYY
AddEdge duplicateEdges x y diffXY1

# List nodes and the outgoing edges
ListNodes duplicateEdges
ListChildren duplicateEdges x
ListChildren duplicateEdges y