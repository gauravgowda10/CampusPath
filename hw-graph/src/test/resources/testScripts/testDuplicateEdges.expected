# Tests that adding the same edges multiple times

# Create graph
created graph duplicateEdges

# Add nodes to duplicateEdges
added node x to duplicateEdges
added node y to duplicateEdges

# Add edges to duplicateEdges
added edge sameXX from x to x in duplicateEdges
added edge sameYY from y to y in duplicateEdges
added edge diffXY1 from x to y in duplicateEdges

# Add Duplicate edges to duplicateEdges
added edge sameXX from x to x in duplicateEdges
added edge sameYY from y to y in duplicateEdges
added edge diffXY1 from x to y in duplicateEdges

# List nodes and the outgoing edges
duplicateEdges contains: x y
the children of x in duplicateEdges are: x(sameXX) y(diffXY1)
the children of y in duplicateEdges are: y(sameYY)